package.json

```diff
@@ -23,41 +27,51 @@
   },
   "homepage": "https://github.com/Chrysalis-Papilio/Soen-490-Papilio-Backend#readme",
   "devDependencies": {
+    "@babel/preset-typescript": "^7.18.6",
     "@types/body-parser": "^1.19.2",
     "@types/bunyan": "^1.8.8",
     "@types/dotenv": "^8.2.0",
     "@types/express": "^4.17.14",
-    "@types/jest": "^29.1.1",
+    "@types/jest": "^29.1.2",
+    "@types/lodash": "^4.14.191",
     "@types/mysql": "^2.15.21",
     "@types/node": "^18.7.23",
     "@types/pg": "^8.6.5",
+    "@types/supertest": "^2.0.12",
     "@typescript-eslint/eslint-plugin": "^5.38.1",
     "eslint": "^8.24.0",
     "eslint-config-standard-with-typescript": "^23.0.0",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-n": "^15.3.0",
     "eslint-plugin-promise": "^6.0.1",
-    "jest": "^29.1.2",
-    "nodemon": "^1.19.4",
+    "jest-serial-runner": "^1.2.0",
+    "jest-stare": "^2.4.1",
+    "nodemon": "^2.0.20",
     "prettier": "^2.7.1",
+    "supertest": "^6.3.0",
+    "ts-jest": "^29.0.3",
     "ts-node": "^10.9.1",
     "tsc-watch": "^5.0.3",
-    "typescript": "^4.8.4"
+    "typescript": "^4.6.3"
   },
```

Comment:
We've been doing development with typescript 4.8.4. Can you switch to that, just for consistency. Thanks